repair/repair.hh

```diff
@@ -204,6 +204,8 @@ public:
     repair_stats _stats;
     bool _row_level_repair;
     uint64_t _sub_ranges_nr = 0;
+    std::unordered_set<sstring> dropped_tables;
+    bool ignore_dropped_tables = false;
```

Comment:
When would we want this to be false?

Reply:
Note it can also refer to a newly added table, if the repair master saw it but the follower hasn't seen it yet.

Reply:
Repair master and followers will use the same schema version choose by the repair master. So the master and follower will see the same schema. 

```
commit 3db136f81e2f70593d95d13a28c65281287c7492
Author: Asias He <asias@scylladb.com>
Date:   Fri Jun 14 09:15:26 2019 +0800

    repair: Use the same schema version for repair master and followers
    
    Before this patch, repair master and followers use their own schema
    version at the point repair starts independently. The schemas can be
    different due to schema change. Repair uses the schema to serialize
    mutation_fragment and deserialize the mutation_fragment received from
    peer nodes. Using different schema version to serialize and deserialize
    cause undefined behaviour.
    
    To fix, we use the schema the repair master decides for all the repair
    nodes involved.
    
    On top of this patch, we could do another step to make sure all nodes
    has the latest schema. But let's do it in a separate patch.
    
    Fixes: #4549
    Backports: 3.1
```

Reply:
When ignore_dropped_tables to false? The regular repair should set this to false. It is fine to fail the regular repair and report to user.

Reply:
It's better not to fail the repair. Different people can be in charge of the schema and repair, they should not have to synchronize in order to manage the schema or repair. In fact either or both can be automation.